src/linelist.jl

```diff
@@ -55,63 +66,55 @@ function get_atoms(molecule)
     el1, el2        
 end
 
+"""
+This type represents an individual line.
+"""
+struct Line{F}
+    wl::F              #given in nm, convert to cm
+    log_gf::F          #unitless
+    species::String    
+    E_lower::F         #eV
+    log_gamma_rad::F   #s^-1
+    log_gamma_stark::F #s^-1
+    log_gamma_vdW::F   #s^-1 (van der Waals)
+end
+
+"""
+Construct a `Line` without explicit broadening parameters.
+"""
+function Line(wl::F, log_gf::F, species::String, E_lower::F) where F
+    e = electron_charge_cgs
+    m = electron_mass_cgs
+    c = c_cgs
+    Line(wl, log_gf, species, E_lower, log10(8π^2 * e^2 / m / c / wl^2) + log_gf, -Inf, -Inf)
```

Comment:
For performance reasons, it might be advisable avoiding the use of `Inf`. I'm pretty sure that there is extra overhead in floating point operations involving `Inf`. Although, I totally understand if you decide that this optimization is not necessary.

If you wanted to do something different, you could potentially:
- introduce a flag to `Line` that indicates whether the broadening parameters are missing
- Store the broadening parameter instead of log10 of the broadening parameter
- You could replace `-Inf` with `nextfloat(log10(floatmin(F)))`. Note that the use of `nextfloat` seeks to avoid evaluation to a [subnormal](https://en.wikipedia.org/wiki/Denormal_number) value when you raise 10 to the power of this value. When `log_gamma_rad` is greater than or equal to -30.4 (-291.3), then this won't introduce any error while using 32-bint floats (64-bit floats)

Reply:
I am probably going to switch from log gamma to gamma in the near future. But also, those infs need to be replaced with a scientifically reasonable approximation no matter what, so they will be gone by the time we are ready to submit.